README.md

```diff
@@ -24,9 +24,13 @@ docker-compose up -d
 
 Once you started stack you can use the following services:
 
-* file storage service: http://localhost/api/file
+* file storage service: http://localhost/api/files
 
-To check service availability you can get its status using `/status` endpoint, e.g.: http://localhost/api/file/status
+To check service availability you can get its status using `/status` endpoint, e.g.: http://localhost/api/files/status
+
+To upload file you can use http://localhost/api/files endpoint sending MultiPartFile and checksum.
```

Comment:
Since we have API Reference page at Wiki now let's simply add a link to it here in README.md there instead of duplicating info.

Reply:
added

Reply:
added API Reference page in README.md